### PR TITLE
Re-adding the baroque rifle (test merge recommended)

### PR DESCRIPTION
**Pull request by Dangerlurking**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

First ever PR and untested locally, sorry for any issues. Undoes Dissent's removal of the baroque in PR #4544 without any changes to the rifle. 
	
<hr>
</details>

## Changelog
:cl:
add: Adds the baroque rifle back.
/:cl:

### CODE REVIEW

**Comment by cdb-is-not-good** [comment]
When I get back from my shift I'll propose some code changes to maybe remedy some of the balance issues this weapon was plagued by.

**Comment by Dangerlurking** [comment]
Open to suggestions. In the Discord I did see some mention of things such as timers to put bullets into the rifle, which I would be fine with implementing. I just want to give the Lodge a fighting chance against bigger enemies, especially since low-pop at the moment forces them to be 100% self-reliant for heavier weapons most of the time.

**Comment by cdb-is-not-good** [comment]
Agree. I'll take a little balance pass over it when I get home, I think making it slower to get off shots would be the biggest boon to balance, maybe couple that with a bit reduced damage and Increased mob damage.

**Comment by Trilbyspaceclone** [comment]
I personally wish the gun was more uniquic then a bolt gun with a different coat of paint but thats something of a when i code it problem
(Something i thought about was having charge powerder that you load in with bullet that deals more damage based on how much you add to the gun. 1 is half damage 2 is full 3 is 2x but risks destorying the gun, and 4+ will break the gun and scale deal damage to you)

File: sojourn-station.dme [hunk 1]
```diff
@@ -2886,6 +2886,7 @@
 #include "code\modules\projectiles\guns\projectile\automatic\triage.dm"
 #include "code\modules\projectiles\guns\projectile\automatic\trouble_shooter.dm"
 #include "code\modules\projectiles\guns\projectile\automatic\vector.dm"
+#include "code\modules\projectiles\guns\projectile\boltgun\baroque.dm"
```
Comment (Trilbyspaceclone) [inline sojourn-station.dme:2889]: Out of order!
make sure to open this in dm and hit complie to fix

**Comment by cdb-is-not-good** [comment]
> I personally wish the gun was more uniquic then a bolt gun with a different coat of paint but thats something of a when i code it problem
> (Something i thought about was having charge powerder that you load in with bullet that deals more damage based on how much you add to the gun. 1 is half damage 2 is full 3 is 2x but risks destorying the gun, and 4+ will break the gun and scale deal damage to you)

Oh I quite like that actually

File: sojourn-station.dme (hunk 1 above)
Comment (Dangerlurking) [reply sojourn-station.dme:2889]: Sorry, what do you mean by "dm"?

Comment (Dangerlurking) [reply sojourn-station.dme:2889]: Nevermind, realized you meant dreammaker, working on it now.

**Comment by ChefDoggo** [comment]
imo because of how comedically unbalanced this gun is it should get changes to it before being readded to crafting and whatnot, the thing literally oneshot gibbed most mobs and wasn't exactly hard to make either. im not a code goblin though so just my take

File: sojourn-station.dme (hunk 1 above)
Comment (Dangerlurking) [reply sojourn-station.dme]: dme has been compiled and re-uploaded, try it now.

**Comment by cdb-is-not-good** [comment]
> imo because of how comedically unbalanced this gun is it should get changes to it before being readded to crafting and whatnot, the thing literally oneshot gibbed most mobs and wasn't exactly hard to make either. im not a code goblin though so just my take

The hunters being a faction of monster-hunters puts them in line with having a gun that is particularly good at killing monsters, It was mostly used for big game like the bears, renders, etc because - yknow. Using it on a normal mob makes it paste.

**Comment by ChefDoggo** [comment]
> > imo because of how comedically unbalanced this gun is it should get changes to it before being readded to crafting and whatnot, the thing literally oneshot gibbed most mobs and wasn't exactly hard to make either. im not a code goblin though so just my take
> 
> The hunters being a faction of monster-hunters puts them in line with having a gun that is particularly good at killing monsters, It was mostly used for big game like the bears, renders, etc because - yknow. Using it on a normal mob makes it paste.

that was the issue though, they were using it on all mobs which made their reason to exist void, you're not hunting for game if you're turning them all to paste hence why it was removed to begin with, it goes against the hunters reason to exist. you can't butcher red mist

**Comment by cdb-is-not-good** [comment]
The point of the hunt isn't for materials, it's for the sake of the hunt. Pasting mobs doesn't really detract from that, though it is funny

**Comment by Dangerlurking** [comment]
In my experience the baroque was more of a rifle to kill big game fast and quickly, those usually being the bears, the great white, and the xenos, all of which are a serious threat in the Lodge's neck of the woods that they inhabit. If you needed to hunt small game with the baroque you were doing hunting wrong to begin with. It's not dissimilar to an 1800s elephant rifle.

**Comment by king5327** [comment]
This does the funny 'changes the entire dme' so I'm not going to merge or test it yet.

**Comment by cdb-is-not-good** [comment]
> This does the funny 'changes the entire dme' so I'm not going to merge or test it yet.

Doesn't that happen anytime you re-enable a file that was previously ticked to off?

**Comment by king5327** [comment]
> > This does the funny 'changes the entire dme' so I'm not going to merge or test it yet.
> 
> Doesn't that happen anytime you re-enable a file that was previously ticked to off?

That should only be one line worth of dme changes, not the full six or so thousand.

**Comment by Dangerlurking** [comment]
> > > This does the funny 'changes the entire dme' so I'm not going to merge or test it yet.
> > 
> > 
> > Doesn't that happen anytime you re-enable a file that was previously ticked to off?
> 
> That should only be one line worth of dme changes, not the full six or so thousand.

Using diffchecker.com to compare the two files the only difference that's in them is the baroque being added to the dme on line 2892. Would it be easier if I PM'd you on Discord to discuss this more?

**Comment by Dangerlurking** [comment]
DME has been adjusted, I thought that copying to the repo and overwriting the old DME would work. I was wrong.

**Pull request merged**